plugins/module_utils/redfish_utils.py

```diff
@@ -63,11 +69,23 @@ def get_request(self, uri):
         req_headers = dict(GET_HEADERS)
         username, password, basic_auth = self._auth_params(req_headers)
         try:
-            resp = open_url(uri, method="GET", headers=req_headers,
-                            url_username=username, url_password=password,
-                            force_basic_auth=basic_auth, validate_certs=False,
+            #To support HPE operations
+            if os.path.isfile(home + "/sessionfile.txt"):
```

Comment:
What exactly is in sessionfile.txt? I see bits of "iLO" information, which doesn't seem appropriate for the standard module. How is this file constructed by users?

Reply:
The sessionfile.txt is temporary persistent storage file storing session id. This is important for managing the session id in between the playbooks or within a single playbook.  If the session is already established, it will use the session id from this file. It can be used by all servers and not just iLO. This file is automatically created when the Login task is executed. The login task will soon be available as part of HPE playbooks.